summary31.txt

```diff
@@ -0,0 +1,100 @@
+Infrastructure Removed radio_tower from list of hardcoded functions
+Interface Adjust martial arts style manual message.
+Balance Simplify armor formula
+Interface Fix makeshift magazine recipe category
+Mods Urban Development Part 3
+Interface Interface fixes
+Balance Fixed martial arts belts json
+Interface Fix chem_sulphur recipe subcategory
+Interface Move weapon recipes to highest damage type
+Interface Redraw description of new selected mod if mod gets removed from active list
+Interface Customize Wield Item menu (and more)
+Features Allow character generation menu to scale to screen size
+Interface Update default font values to prevent tiny overmap font usage on new game installations.
+Performance Update bitrotted gain doubling for working off skill rust
+Bugfixes Remove the null char keybinding to debug mode
+Interface Removed messages from artifact smoke generation
+Content Crafting soap out of lye made from ash
+Infrastructure Display float options in the correct decimal format
+Interface Fix invlet disappearing when choosing to wear an item when unwielding it
+Balance Updated .50 BMG recoil to match kinetic energy.
+Balance Removed rarity node from zweihander_inferior
+Bugfixes Attempt to fix wielding
+Infrastructure Store creatures in shared pointers
+Balance Add withered plant as a random underbush loot
+Infrastructure Jsonify mapgen (churches and doctor office)
+Infrastructure Prison mapgen JSONify third try
+Infrastructure Remove player::can_wield from Lua bindings
+Features Allow martial arts to force "offhand" unarmed strikes (kicks etc.) when wielding weapons
+Infrastructure Turn some hardcoded modifiers into effects
+Bugfixes In the character creation screen, fix crash dwhen skill list is larger than skill window height
+Balance Make vibrators usable without batteries
+Content Micro Atomic Plant for Bright Nights mod
+Infrastructure Ranged balance unit test
+Bugfixes Fix misc keybinding menu issues
+Balance Reduced the craft time of the crowbar
+Bugfixes Fix crash when attempting to throw items, fix interface for guns being thrown
+Infrastructure Include LUA_BINARY in LUA_CANDIDATES
+Infrastructure Shrink git checkout depth for faster builds
+Balance Make inwoods runs less bland
+Interface Storage Battery Description Change
+Infrastructure Remove redunant function game::monster_at
+Infrastructure Replace most usage of game::mon_at with game::critter_at
+Balance Fix some tools recipes to match output item volume.
+Balance Brass catcher catches ammo links
+Interface Misc changes to improve uimenu
+Interface Keep new item's invlet if original stack does not have one
+Infrastructure Low hanging astyling, the return of the revenge.
+Interface Alert about book required intelligence when needed
+Bugfixes Small changes in AIM
+I128 Translate missed message
```

Comment:
```suggestion
I18N Translate missed message
```

Reply:
Change `I128` to `I18N` here and below.

Reply:
Taken from pr line and propagated due to auto complete... will fix.